quickstart/2-browser-parallel-computation/aqua/getting-started.aqua

```diff
@@ -0,0 +1,60 @@
+-- Import builtin services
+import "@fluencelabs/aqua-lib/builtin.aqua"
+-- Import subnets
+import Subnet, Worker from "@fluencelabs/aqua-lib/subnet.aqua"
+
+-- Deal id of the deployed worker. We need this to resolve subnet 
+const ADDER_DEAL_ID ?= "0x3C85E6765e58Df1e6c09E5CDC0C386bEEBdBa155"
+
+-- The service runs on a Fluence node
+service Adder("adder"):
+    add_one(value: u64) -> u64
+
+-- Function to get all workers from subnet
+func resolveSubnet() -> []Worker:
+    -- It's possible to resolve subnet only on HOST_PEER_ID i.e. on relay. This won't work locally.
+    on HOST_PEER_ID:
+        -- Resolving subnets using Deal id of deployed service
+        subnet <- Subnet.resolve(ADDER_DEAL_ID)
+    <- subnet.workers
+
+-- This structure forms a computation requests from frontend
```

Comment:
```suggestion
This data structure is used to represent functions ares such as provided by the frontend, i.e., browser.
```

Reply:
Changed it to
`-- This data structure is representing calling aqua function on the worker via frontend.`

Reply:
close :) 
`This data structure represents the aqua function that is calling the worker via the frontend.`